simtools/simtel/simtel_histograms.py

```diff
@@ -92,7 +92,7 @@ def _combine_histogram_files(self):
                 for o in yield_toplevel_of_type(f, Histograms):
                     try:
                         hists = o.parse()
-                    except Exception:
+                    except Exception:  # pylint: disable=broad-except
```

Comment:
I disabled the pylint error because there are lots of exceptions that could occur when we parse and I don't think we can catch them individually. Also, it doesn't really matter for the use case. It doesn't matter why a file cannot be read, we should skip it regardless.

Reply:
Agree.